### PR TITLE
Added persistent flags applayer tag for root

### PR DESCRIPTION
**Pull request by Midnight-cahq**

### CODE REVIEW

File: glue.go [hunk 1]
```diff
@@ -73,6 +74,8 @@ const (
 
 	// TagCliCommand is tag to mark exported cli commands.
 	TagCliCommand = "cli.cmd"
+
```
Comment (snovichkov) [inline glue.go:77]: Please add comment

File: glue.go [hunk 2]
```diff
@@ -73,6 +74,8 @@ const (
...
+	TagRootPersistentFlags = "cli.persistentFlags"
```
Comment (snovichkov) [inline glue.go]: Use underscore with dot separators

File: glue.go [hunk 3]
```diff
@@ -199,20 +202,29 @@ func (k *app) initContainer() error {
 					},
 				}
 
+				rootCmd.PersistentFlags()
```
Comment (snovichkov) [inline glue.go]: What for?

Comment (Midnight-cahq) [reply glue.go]: removed

File: glue.go (hunk 2 above)
Comment (Midnight-cahq) [reply glue.go]: fixed

File: glue.go (hunk 1 above)
Comment (Midnight-cahq) [reply glue.go:77]: added

**Pull request merged**